programs/clearing_house/src/lib.rs

```diff
@@ -479,10 +480,10 @@ pub mod clearing_house {
         let now = clock.unix_timestamp;
 
         let remaining_accounts_iter = &mut ctx.remaining_accounts.iter().peekable();
-        let _oracle_map = OracleMap::load(remaining_accounts_iter, clock.slot)?;
+        let mut oracle_map = OracleMap::load(remaining_accounts_iter, clock.slot)?;
         let bank_map = BankMap::load(&get_writable_banks(bank_index), remaining_accounts_iter)?;
 
-        let market_map = MarketMap::load(
+        let mut market_map = MarketMap::load(
```

Comment:
should use `get_writable_markets_for_user_positions` to get the writable markets needed for the update_amms function (should does this in the other ix that call update_amms as well)